pex/venv/installer.py

```diff
@@ -549,6 +549,7 @@ def _populate_sources(
         dst=dst,
         exclude=(
             "__main__.py",
+            "__pex__",
```

Comment:
A venv PEX never needed the magic `__pex__` import hook, which won't work anyhow since a venv PEX has no Pex .bootstrap code carrying the import hook implementation code.